### DISCUSSION TITLE
Is there a way to achieve this?

### DISCUSSION THREAD

**Discussion by Kokecena** [jxlsteam/jxls#301] [Q&A]
Hello, first of all, excellent library, I just found it and I am using it for an internal project at my work.

I am trying to make an excel document with different spreadsheets for all the days of the month, using this format

here's my template:
![image](https://github.com/jxlsteam/jxls/assets/75457600/c19eee47-1ea6-4479-8ed2-66f9820129e7)

I was guided by the example provided in the [each](https://jxls.sourceforge.net/each.html) section.
and I would like to achieve the spreadsheets by date, as shown in the multisheet section

but I can't figure out how to generate it, my information is like this

`List<Employees>`

`Employees` has `List<Employee> cachedEmployees` and `String date`

and in `Employee` there are all the attributes that are seen in the template

Following the example I assumed it would be something like
```
data.put("employees",lists.stream().map(Employees::getCachedEmployees).toList());
data.put("titleSheet",lists.stream().map(Employee::getDate).toList());
```
but obviously it doesn't work, so it generated an excel for each day, it works for me for now, but I would like to know if it is possible to achieve what I mention.

![image](https://github.com/jxlsteam/jxls/assets/75457600/2ef2bdf0-fdc5-4d1d-935e-b55e92efb873)

this is my code
![image](https://github.com/jxlsteam/jxls/assets/75457600/83e58500-bb07-473a-ac48-e21689fdb139)

thanks and sorry for my bad english :p

**Comment by SoltauFintel** [comment]
The multisheet attribute in jx:each is missing. jx:each(multisheet="sheetnames" ...)

data.put("sheetnames", <List with "1", "2", "3", ... "31">);

**Reply by Kokecena** [reply]
I really appreciate that you answered me.

And yes, that is what I had understood from the documentation for multi-spreadsheets and if I did it, here is the code.

![image](https://github.com/jxlsteam/jxls/assets/75457600/99e8868e-27b3-4ef1-82fd-228eee922413)

This is the result I expected
![image](https://github.com/jxlsteam/jxls/assets/75457600/e925c187-bed3-4a30-809e-059c721b643d)
![image](https://github.com/jxlsteam/jxls/assets/75457600/a92455c1-94b5-499b-a2e9-862dd3bc4064)

Instead what I got was this xD
![image](https://github.com/jxlsteam/jxls/assets/75457600/32a20d27-fe57-42f6-9e7f-966153b5bf86)

![image](https://github.com/jxlsteam/jxls/assets/75457600/6f5798bf-83ae-4eb7-8ba7-7383daabbcce)

This is my "Empleados" class, basically the employee incidents are received from the repository and a list of "Empleados" is created that contains the list of "Empleado" and the date attribute of the incidents of the day
![image](https://github.com/jxlsteam/jxls/assets/75457600/7fc5d861-3c94-45c1-b5a3-3b0b28498b26)
![image](https://github.com/jxlsteam/jxls/assets/75457600/62d4d5d0-ccd2-4d20-8507-be680908bc87)

I was wondering how I could achieve this because the class I use to contain the employees is the same list, that's why my solution, the one I put at the beginning, was to generate an excel file for each day and my goal was to fill everything in a single excel

Sorry if I was very explicit, I just felt that I didn't explain very well what I wanted to achieve.

In any case, if it is not possible, there is no problem.

Thank you for your help and attention

**Reply by SoltauFintel** [reply]
Is there a getArea() ?

**Reply by SoltauFintel** [reply]
I can probably only help if you provide a Github repository with an example.

Maybe I don't even understand what the problem is.

**Reply by SoltauFintel** [reply]
Try to change "private Area area" to "private String area" + getter. I'm not sure if Area works. I use JXLS with maps and not with classes.

**Comment by SoltauFintel** [comment]
One thing I forgot to mention: please remove `sheetStreaming="true"`. I recommend that newbies don't use any optimization until everything works.
